src/documentation/print-linting-and-testing-wiki.ts

```diff
@@ -0,0 +1,186 @@
+import { setMinLevelOfAllLogs } from '../../test/functionality/_helper/log';
+import { LogLevel } from '../util/log';
+import { codeBlock } from './doc-util/doc-code';
+import { FlowrDockerRef, FlowrGithubBaseRef, FlowrSiteBaseRef, getFilePathMd, RemoteFlowrFilePathBaseRef } from './doc-util/doc-files';
+
+function getText() {
+	return `
+For the latest code-coverage information, see [codecov.io](https://app.codecov.io/gh/flowr-analysis/flowr), 
+for the latest benchmark results, see the [benchmark results](${FlowrSiteBaseRef}/wiki/stats/benchmark) wiki page.
+
+- [Testing Suites](#testing-suites)
+  - [Functionality Tests](#functionality-tests)
+    - [Test Structure](#test-structure)
+    - [Writing a Test](#writing-a-test)
+    - [Running Only Some Tests](#running-only-some-tests)
+  - [Performance Tests](#performance-tests)
+  - [Oh no, the tests are slow](#oh-no-the-tests-are-slow)
+  - [Testing Within Your IDE](#testing-within-your-ide)
+    - [Using Visual Studio Code](#vs-code)
+    - [Using WebStorm](#webstorm)
+- [CI Pipeline](#ci-pipeline)
+- [Linting](#linting)
+  - [Oh no, the linter fails](#oh-no-the-linter-fails)
+  - [License Checker](#license-checker)
+
+## Testing Suites
+
+Currently, flowR contains two testing suites: one for [functionality](#functionality-tests) and one for [performance](#performance-tests). We explain each of them in the following.
+A more general test experience is granted by \`npm run checkup\` which will include the construction of the docker image, the generation of the wiki pages, and the linter.
```

Comment:
I think this sentence from me is still a bit.. blunt. Can you add a helping sentence in between?